ci_microshift.sh

```diff
@@ -14,7 +14,6 @@ export CRC_ZSTD_EXTRA_FLAGS="-10"
 
 # Delete the crc domain which created by snc so it can created
 # for crc test
-sudo virsh destroy crc
```

Comment:
did you consider ignoring the error (`|| true`)?, so in case of an unsuccessful run and if the VM stays running, it'll stop that

Reply:
if it unsuccessful then bundle will be not generated so there is no point to run next steps.

Reply:
got it, if creatdisk.sh fails then bundle will not be generated and this steps will not even run